A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.html

```diff
@@ -13,7 +13,7 @@
             </g>
             <g id="sts-reminder">
                 <rect x="350" y="572" width="50" height="20" fill="black" stroke="white" stroke-width="2" />
-                <text x="375" y="588.5" fill="white" font-size="20px" text-anchor="middle">STS</text>
+                <text x="375" y="588.5" fill="white" style="font-size: 12pt;" text-anchor="middle">STS</text>
```

Comment:
**question**: The _point_ (pt) unit is for printing. Are you sure you want to use it?

Reply:
I think it is not. But this just temporary fix(because Lower ECAM Status page also had that problem). We will rewrite every instrument to react isn’t it?

Reply:
It's not really a matter of opinion 😉. You can refer to [this documentation](https://www.w3.org/Style/Examples/007/units.en.html) for more information.

Reply:
oh what the fuck why i am using pt?

Reply:
sorry i fix that